src/neptune_pytorch/impl/__init__.py

```diff
@@ -13,25 +13,79 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-__all__ = ["__version__", "NeptuneCallback"]
+__all__ = ["__version__", "NeptuneLogger"]
 
-from typing import Optional
+from typing import (
+    Optional,
+    Union,
+)
 
 import torch
 
 from neptune_pytorch.impl.version import __version__
 
+try:
+    # neptune-client>=1.0.0 package structure
+    from neptune import Run
+    from neptune.handler import Handler
+    from neptune.internal.utils import verify_type
+except ImportError:
+    # neptune-client=0.9.0+ package structure
+    from neptune.new import Run
+    from neptune.new.handler import Handler
+    from neptune.new.integrations.utils import verify_type
+
 IS_TORCHVIZ_AVAILABLE = True
 try:
     import torchviz
 except ImportError:
     IS_TORCHVIZ_AVAILABLE = False
 
+INTEGRATION_VERSION_KEY = "source_code/integrations/neptune-pytorch"
+
 
 class NeptuneLogger:
+    """Captures model training metadata and logs them to Neptune.
+
+    Note: To use this module, you need to have PyTorch installed.
```

Comment:
Maybe this is not necessary, as in this case it should be clear that it's about PyTorch :)